Content.Server/_Goobstation/Speech/Components/CheeseAccentComponent.cs

```diff
@@ -0,0 +1,8 @@
+using Content.Server.Speech.EntitySystems;
+
+namespace Content.Server.Speech.Components;
```

Comment:
For when you refactor this to goobmod.
```suggestion
namespace Content.Goobstation.Common.Speech;